sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizeEntitiesAsync.java

```diff
@@ -27,11 +27,12 @@ public static void main(String[] args) {
         String document = "Satya Nadella is the CEO of Microsoft";
 
         client.recognizeEntities(document).subscribe(
-            entity -> System.out.printf(
-                "Recognized categorized entity: %s, entity category: %s, entity sub-category: %s, score: %f.%n",
-                entity.getText(), entity.getCategory(), entity.getSubCategory(), entity.getConfidenceScore()),
+            entityCollection -> entityCollection.forEach(entity -> System.out.printf(
```

Comment:
Could u help me understand why before we didn't need to do a foreach with entity, but now we do have to do a foreach on entityCollection?

Reply:
Previewly, we returned TextAnalyticsPagedFlux<CategorizedEntity> which can directly getting the CategorizedEntity. With Current design, Mono<CategorizedEntityCollection>,  we are getting CategorizedEntityCollection object first, but not CategorizedEntity